tutorials/cp-mobile-dev-kit-add-validation/cp-mobile-dev-kit-add-validation.md

```diff
@@ -144,11 +144,28 @@ Deploy the updated application to your MDK client.
 
 [OPTION END]
 
+[OPTION BEGIN [Web]]
+
+1. Either click the highlighted button or refresh the web page to load the changes.
+
+    !![MDK](img_5.3.png)
+
+    >If you see the error `404 Not Found: Requested route ('xxxxxtrial-dev-nsdemosampleapp-approuter.cfapps.xxxx.hana.ondemand.com') does not exist.` while accessing the web application, make sure that in your space cockpit, highlight applications are in started state.
+
+    >!![MDK](img_5.6.png)
+
+2. Update a record with no contain of **@** in Email property throws a validation failure message.
```

Comment:
Can you please rephrase to make it better understandable? Grammar seems to be a bit off too.